### PR TITLE
chore(yarn): move to yarn

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
+# misc
 .parrot
+package-lock.json
```
Comment (Francois-Esquire) [inline .gitignore:3]:
```suggestion
```

Comment (Francois-Esquire) [reply .gitignore:3]: We wouldn't need this I think, with the `package-lock=false` in the `.npmrc`

Comment (JAdshead) [reply .gitignore:3]: this would reduce the chance of package-lock being committed by mistake, better to be explicit

**Review by JAdshead — COMMENTED** [review]
are you sure that the versions in the yarn lock are the same as set in package-lock ?

**Comment by JamesSingleton** [comment]
> are you sure that the versions in the yarn lock are the same as set in package-lock ?

@JAdshead Checking the `dependencies`, yes. I will need to update some of the deps as well.

**Review by mtomcal — DISMISSED** [review]

**Review by mtomcal — APPROVED** [review]

**Review by nellyk — APPROVED** [review]

**Pull request merged**